sonar-db/src/main/java/org/sonar/db/user/RoleDao.java

```diff
@@ -32,14 +32,27 @@
     return mapper(dbSession).selectComponentIdsByPermissionAndUserId(permission, userId);
   }
 
+  /**
+   * @deprecated replaced by {@link org.sonar.db.permission.GroupPermissionDao#selectGroupPermissions(DbSession, long, Long)}
+   * and {@link org.sonar.db.permission.GroupPermissionDao#selectAnyonePermissions(DbSession, Long)}
+   */
+  @Deprecated
   public List<String> selectGroupPermissions(DbSession session, String groupName, @Nullable Long resourceId) {
     return session.getMapper(RoleMapper.class).selectGroupPermissions(groupName, resourceId, DefaultGroups.isAnyone(groupName));
   }
 
+  /**
+   * @deprecated replaced by {@link org.sonar.db.permission.GroupPermissionDao#insert(DbSession, GroupPermissionDto)}
+   */
+  @Deprecated
   public void insertGroupRole(DbSession session, GroupPermissionDto dto) {
     mapper(session).insertGroupRole(dto);
   }
 
+  /**
+   * @deprecated does not support organizations on anyone groups
+   */
+  @Deprecated
   public void deleteGroupRole(GroupPermissionDto groupRole, DbSession session) {
```

Comment:
![INFO](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-info.png) Do not forget to remove this deprecated code someday. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://dory.sonarsource.com/sonarqube/coding_rules#rule_key=squid%3AS1133)